src/chains-config/logionControllers.ts

```diff
@@ -0,0 +1,36 @@
+import { ControllerConfig } from '../types/chains-config';
+
+/**
+ * Controllers that Sidecar will always default to. This will always be
+ * the optimal controller selection for Polkadot and Kusama.
+ */
+export const logionControllers: ControllerConfig = {
+	controllers: [
+		'AccountsAssets',
+		'AccountsBalanceInfo',
+		'AccountsStakingInfo',
+		'AccountsStakingPayouts',
+		'AccountsVestingInfo',
+		'Blocks',
+		'BlocksExtrinsics',
+		'NodeNetwork',
+		'NodeTransactionPool',
+		'NodeVersion',
+		'PalletsAssets',
+		'PalletsStakingProgress',
+		'PalletsStorage',
+		'Paras',
```

Comment:
Above controller is probably useless currently, but that's a detail.

Reply:
Removed.